docs/learn/learn-jam-faq.md

```diff
@@ -0,0 +1,163 @@
+purpose. With JAM, groundbreaking scalability currently only seen through rollups is brought to the
+consensus layer. Developers no longer need to decide if they build appchains or smart contracts;
+they need a flexible and rich environment for both. So, in short, there will be L2 scalability
+without the need for rollup solutions while being fully flexible to build any application.
+
+## What are services?
+
+Services are modules that run on top of JAM. One of these services would be, for example, the
+ChainService. This service would implement the parachain logic that is currently enshrined in the
+Polkadot protocol. JAM’s `refine` and `accumulate` phases are generic and provided by these
+services. This will make it possible to implement even more completely permissionless services, such
+as providing an actor-like framework to run on top of JAM.
+
+## How does JAM differ from traditional blockchain networks?
+
+JAM is built as a distributed computer. This means that it can run almost any task that can be
+expressed as a service. The JAM chain will have almost no functionality, meaning there will be no
+user transactions. All the logic, like governance, staking, etc., would live on system **services**.
+JAM will provide synchronous composability across heterogeneous services, enabling new kinds of
+interoperability.
+
+## Will Agile Coretime remain part of the Polkadot vision?
+
+Yes. [Agile Coretime](./learn-agile-coretime.md), an innovation that transforms Polkadot into a
+computational resource, will be rolled out onto the Polkadot network in the near future. JAM is
+further down the line, but Coretime will similarly be available to purchase on JAM.
+
+## How do services interact with JAM?
+
+Services interact with JAM through predefined entry points: Refine, Accumulate, and onTransfer (see
+more information on the [JAM Chain page](./learn-jam-chain.md#service-entry-points)). Each service
+defines its specific functions and workflows, executed within the JAM framework according to the
+network's requirements.
+
+## What are the main components of JAM?
+
+JAM consists of four main components: Refine, Accumulate, and onTransfer (see more information on
+the [JAM Chain page](./learn-jam-chain.md#service-entry-points)). These components define the
+workflow for processing and integrating data within the network.
+
+## What is the goal of JAM?
+
+JAM's primary goal is to provide a flexible and efficient framework for managing data and
+computations within a network. It aims to streamline data integration and maintenance while ensuring
+the network's integrity and security.
+
+## Is JAM related to Polkadot?
+
+Yes, JAM is a potential candidate for evolving the Polkadot Relay Chain. The key change is that
+parachains are no longer enshrined; anyone can permissionlessly run their service. One service will
+be, for example, the current parachain functionality.
```

Comment:
Are we implying that parachain protocol is no longer enshrined like they are currently on the relay chain and will be running on top of a service that is compatible with parachain protocol? In that case, we will have to rephrase the answer better.

> anyone can permissionlessly run their service.

Anyone can permissionlessly run which service? Won't parachain protocol be running as a system service?